### ISSUE TITLE
404 when trying to build

### ISSUE THREAD

**Issue by casept** [ipfs/ipget#47]
I get a 404 and subsequent build failure when following the instructions for installing from source.
```
david@t400:~/go/src/github.com/ipfs/ipget$ make install 
installing gx v0.11.0
==> Using wget to download "https://ipfs.io/ipfs/QmNZL8wNsvAGdVYr8uGeUE9aGfHjFpHegAWywQFEdSaJbp/gx/v0.11.0/gx_v0.11.0_linux-amd64.tar.gz" to "bin/tmp/gx.tar.gz"
--2017-05-09 21:59:57--  https://ipfs.io/ipfs/QmNZL8wNsvAGdVYr8uGeUE9aGfHjFpHegAWywQFEdSaJbp/gx/v0.11.0/gx_v0.11.0_linux-amd64.tar.gz
Resolving ipfs.io (ipfs.io)... 104.236.151.122, 178.62.61.185, 104.236.176.52, ...
Connecting to ipfs.io (ipfs.io)|104.236.151.122|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-05-09 21:59:58 ERROR 404: Not Found.

failed to download https://ipfs.io/ipfs/QmNZL8wNsvAGdVYr8uGeUE9aGfHjFpHegAWywQFEdSaJbp/gx/v0.11.0/gx_v0.11.0_linux-amd64.tar.gz to bin/tmp/gx.tar.gz
Makefile:18: recipe for target 'bin/gx-v0.11.0' failed
make: *** [bin/gx-v0.11.0] Error 1
```

**Comment by noffle** [comment]
Thanks for catching this, David. It looks like the `gx` version referenced isn't present in the `dist_root` hash in the Makefile.

Fixed as of `v0.2.5`. Try fetching the latest `master` and `make install`ing again.

**Comment by casept** [comment]
Works now, thank you for the prompt fix!
